core/authentication_api.php

```diff
@@ -717,6 +717,20 @@ function auth_does_password_match( $p_user_id, $p_test_password ) {
 	}
 
 	$t_password = user_get_field( $p_user_id, 'password' );
```

Comment:
For readability, we should probably call rename `$t_password` to `$t_password_hash`.